src/framework/components/light/light_data.js

```diff
@@ -14,6 +14,7 @@ pc.extend(pc, function () {
         this.innerConeAngle = 40;
         this.outerConeAngle = 45;
         this.falloffMode = pc.LIGHTFALLOFF_LINEAR;
+        this.mask = 1;
```

Comment:
Oh, actually, I see you do save the mask property into the light. But mask is not initialized in the pc.Light constructor. I think that's my problem with it. :)